packages/cart/src/types/shipping-method.ts

```diff
@@ -0,0 +1,16 @@
+export interface CreateShippingMethodDTO {
+  name: string
+
+  cart_id?: string
+
+  amount: number
+  data?: Record<string, unknown>
+}
+
+export interface UpdateShippingMethodDTO {
```

Comment:
question: if the cart id is optional on create, should we add it to the update dto to be able to set it?